token/program-2022/src/extension/confidential_transfer/processor.rs

```diff
@@ -157,7 +160,8 @@ fn process_configure_account(
 
         This should just be encoded as [0; 64]
     */
-    confidential_transfer_account.pending_balance = EncryptedBalance::zeroed();
+    confidential_transfer_account.pending_balance_lo = EncryptedBalance::zeroed();
+    confidential_transfer_account.pending_balance_hi = EncryptedBalance::zeroed();
```

Comment:
Update comment preceding this?

Reply:
Thanks! Updated now.

Reply:
I don't see this change either

Reply:
Ah terribly sorry. The changes should be pushed now.